clients/metadata/metadata_cassandra.go

```diff
@@ -726,6 +726,12 @@ func (s *CassandraMetadataService) UpdateDestination(ctx thrift.Context, updateR
 	if !updateRequest.IsSetChecksumOption() {
 		updateRequest.ChecksumOption = common.InternalChecksumOptionPtr(existing.GetChecksumOption())
 	}
+	isMultiZone := existing.GetIsMultiZone()
+	if !updateRequest.IsSetZoneConfigs() {
+		updateRequest.ZoneConfigs = existing.GetZoneConfigs()
+	} else {
+		isMultiZone = true
```

Comment:
That's right, we disallow single zone config in CLI part. Zone config(s) are designed for multi-zone only.